constants/serverPackets.py

```diff
@@ -159,13 +159,32 @@ def userLogout(userID: int) -> bytes:
 )
 
 
-async def userPanel(userID: int, force: bool = False) -> bytes:
-    if userID == CHATBOT_USER_ID:
+async def userPanel(
+    *,
+    user_id: int | None = None,
+    token_id: str | None = None,
+    allow_restricted_tokens: bool = False,
+) -> bytes:
+    if user_id is not None:
+        # Lookup by user id; assuming the caller wants primary token
+        userToken = await osuToken.get_primary_token_by_user_id(user_id)
+        if userToken is None:
+            # (If no primary token exists, fallback to any token)
```

Comment:
We fallback to non-primary support here to support sending panels/stats for times when there are only non-primary/tourney tokens connected